contracts/src/v0.8/interfaces/IVRFCoordinatorV2Plus.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+// Interface for initial version of VRFCoordinatorV2Plus
+// Functions in this interface may not be supported when VRFCoordinatorV2Plus is upgraded to a new version
```

Comment:
I think we need to commit to support backwards compatibility for `requestRandomWords` and `rawFulfillRandomness`. 

I'm not sure if we should support the same for other functions (e.g. subscription related functions) because it will limit types of improvements that we can make in future versions. If we decide not to support backwards compatibility for other functions, we need a way to clearly tell customers (e.g. comments, subscription management contract only for V1)

Reply:
Although it's hard to predict what features we will introduce in the future, I think `getRequestConfig()` and `getSubscription()` are more likely to change than `createSubscription()`, `requestSubscriptionOwnerTransfer()`, `acceptSubscriptionOwnerTransfer()`, `addConsumer()`, `removeConsumer()` and `cancelSubscription()` and `pendingRequestExists()`